layouts/partials/blocks/awards.html

```diff
@@ -0,0 +1,269 @@
+                <div class="award-entity" data-year="{{ .year }}">
+                  {{ .entity }}:
+                  <span class="award-category">{{ .award_category }}</span>
+                  <p class="award-story">
+                    {{ if .link }}
+                      <a
+                        href="{{ .link }}"
+                        class="award-story-link"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        {{ .story }}
+                      </a>
+                    {{ else }}
+                      {{ .story }}
+                    {{ end }}
+                  </p>
+                </div>
+              {{ end }}
+            </div>
+
+            <p class="award-national-state state-awards-header">State Awards</p>
+            <div class="state-awards">
+              {{ range where (where . "national_or_state" "State") "winner_or_finalist_honorable_mention" "Winner" }}
+                <div class="award-entity" data-year="{{ .year }}">
+                  {{ .entity }}:
+                  <span class="award-category">{{ .award_category }}</span>
+                  <p class="award-story">
+                    {{ if .link }}
+                      <a
+                        href="{{ .link }}"
+                        class="award-story-link"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        {{ .story }}
+                      </a>
+                    {{ else }}
+                      {{ .story }}
+                    {{ end }}
+                  </p>
+                </div>
+              {{ end }}
+            </div>
+
+            <p class="award-national-state state-finalists-header">
+              State Finalists
+            </p>
+            <div class="state-finalists">
+              {{ range where (where . "national_or_state" "State") "winner_or_finalist_honorable_mention" "Finalist/Honorable Mention" }}
+                <div class="award-entity" data-year="{{ .year }}">
+                  {{ .entity }}:
+                  <span class="award-category">{{ .award_category }}</span>
+                  <p class="award-story">
+                    {{ if .link }}
+                      <a
+                        href="{{ .link }}"
+                        class="award-story-link"
+                        target="_blank"
+                        rel="noopener noreferrer"
+                      >
+                        {{ .story }}
+                      </a>
+                    {{ else }}
+                      {{ .story }}
+                    {{ end }}
+                  </p>
+                </div>
+              {{ end }}
+            </div>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+{{ end }}
+
+
+<style>
```

Comment:
I'd like to get these turned into Tailwind. Instead of doing the first-child/last-child margin 0 and having to use 7.5px of margin, it would probably be easier to have `flex flex-col gap-[15px]`.